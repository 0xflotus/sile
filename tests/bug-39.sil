\begin[papersize=a5,class=book]{document}
\script[src=packages/frametricks]
\right-running-head{First Line of Header \break \skip[height=8mm] Second Line of Header}
\showframe[id=all]

First page text
\break \skip[height=22cm]
Second page text

\end{document}
