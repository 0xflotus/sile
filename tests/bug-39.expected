<tests/bug-39.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	54.82079
My 	75.67306
Set font 	Gentium;10;200;regular;normal;;LTR
T	38 73 82 83 84	(First)
Mx 	76.24533
T	80 65 71 69	(page)
Mx 	98.29976
T	84 69 88 84	(text)
Mx 	54.82079
My 	28.99828
T	38 73 82 83 84	(First)
Mx 	76.24291
T	44 73 78 69	(Line)
Mx 	96.51756
T	79 70	(of)
Mx 	107.32445
T	40 69 65 68 69 82	(Header)
Mx 	54.82079
My 	64.67545
T	51 69 67 79 78 68	(Second)
Mx 	86.98391
T	44 73 78 69	(Line)
Mx 	107.25739
T	79 70	(of)
Mx 	118.06309
T	40 69 65 68 69 82	(Header)
[1] Mx 	195.46105
My 	519.80194
T	17	(1)
New page
Mx 	78.73386
My 	76.62033
T	51 69 67 79 78 68	(Second)
Mx 	110.89941
T	80 65 71 69	(page)
Mx 	132.95265
T	84 69 88 84	(text)
[2] Mx 	219.37413
My 	519.75311
T	18	(2)
End page
Finish

